sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileSystemClient.java

```diff
@@ -114,6 +116,21 @@ public DataLakeDirectoryClient getDirectoryClient(String directoryName) {
             blobContainerClient.getBlobClient(directoryName).getBlockBlobClient());
     }
 
+    /**
+     * Initializes a new DataLakeDirectoryClient object by concatenating "" to the end of
```

Comment:
Just so it stylizes the `""` differently.

```suggestion
     * Initializes a new DataLakeDirectoryClient object by concatenating {@code ""} to the end of
```

Reply:
Done